swagger.json

```diff
@@ -4163,6 +4163,12 @@
                     "description": "The address of the job destination, as it would be recognized if provided to maps.google.com",
                     "example": "123 Main St, Philadelphia, PA 19106"
                 },
+                "destination_address_id": {
```

Comment:
Ah, I think you've added this to `DispatchJobCreate`. We do eventually want this to be here, but currently since we don't have a POST endpoint done, maybe we should wait to merge this after that's implemented. 

Also small nit, to be consistent with the descriptions for destination name and destination address, maybe say "The ID of the job destination associated with an address book entry."

Reply:
Moved per our conversation.